package.json

```diff
@@ -99,6 +99,38 @@
                 "command": "makefile.launchRun",
                 "title": "%makefile-tools.command.makefile.launchRun.title%"
             },
+            {
```

Comment:
These commands are intentionally omitted from `package.json` so that they do not appear in the Command Palette. These commands look like the one we exposed for Command Substitution.  They actually don't need to be declared in `package.json`.

Reply:
I was about to comment about the same thing. Otherwise, the change in launch.ts looks good.

Reply:
The changes for those commands have been reverted. 👍